### DISCUSSION TITLE
[Feature Suggestion]: Allow attribute location `method` on expression-bodied properties

### DISCUSSION THREAD

**Discussion by ero-qt** [dotnet/csharplang#7282] [General]
## Summary

This proposal is about allowing the following syntax;
```cs
[method: SomeAttribute]
public string Str => "";
```

## Background and Motivation

We are able to add attributes on both `get` and `set` accessors of a property. Similarly, the above would apply the attribute on `get_Str`.

**Comment by ufcpp** [comment]
https://github.com/dotnet/csharplang/discussions/887
